MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,141 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
 {% endhighlight %}
 {% endtabs %}
 
+## Customize the form fields programmatically
+
```

Comment:
Still in one of the places only the color is mentioned
<img width="520" alt="image" src="https://github.com/user-attachments/assets/54a66d20-1748-43d4-8325-0031b0ea859a" />

It is better to have a common term instead of mentioning every time all the fields
<img width="481" alt="image" src="https://github.com/user-attachments/assets/9896b2ee-3870-44f8-8a0f-723e099d205f" />